### ISSUE TITLE
podman commit: duplicate cid emitted to stdout

### ISSUE THREAD

**Issue by edsantiago** [projectatomic/libpod#705]
Regression: this used to work fine in a recent build; I can try to track down details on request.
```
# podman run --name foo fedora date
Tue May  1 16:07:46 UTC 2018
# podman commit --quiet foo myimg
e05103626afc5df15f986fd9cea774557432a0c7c2b0608275b2fc9e899659c6

e05103626afc5df15f986fd9cea774557432a0c7c2b0608275b2fc9e899659c6
```
That is: CID, blank line, CID again. Both are emitted to stdout, making it impossible (or challenging) to capture the output:
```
# cid=$(podman commit --quiet foo myimg)
# echo $cid
deaa085a491d3362feeb293e14495c9578bd39e652a482d6ef2769b11524be05 deaa085a491d3362feeb293e14495c9578bd39e652a482d6ef2769b11524be05
```

podman-0.4.4-2.git9924956.fc27.x86_64

**Comment by mheon** [comment]
Buildah is printing the image ID as well in `Commit()`. We need to patch this to accept a writer - it's not OK for libraries to unconditionally print to stdout.

**Comment by mheon** [comment]
Opened https://github.com/projectatomic/buildah/pull/647 to address this in Buildah. Once that's merged, we can revendor on the Podman side.

**Comment by mheon** [comment]
Should have been fixed by Buildah revendor in #715
